### PR TITLE
Compute sirna-treated cell genotype probabilities with updated model

### PR DESCRIPTION
**Pull request by MattsonCam**
In this pr, the updated nf1 model is used to compute genotype probabilities of siRNA-treated cells from plate 4 morphology features. Please let me know if you have any suggestions.

### CODE REVIEW

File: 2.evaluate_models/classify_genotypes/nbconverted/plate_4_sirna_genotype_probabilities.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+
+plate4df_path = pathlib.Path(root_dir / "nf1_painting_repo/3.processing_features/data/single_cell_profiles/Plate_4_sc_feature_selected.parquet").resolve(strict=True)
+plate4df = pd.read_parquet(plate4df_path)
+
+data_path = pathlib.Path(root_dir / "1.train_models/classify_genotypes/data").resolve(strict=True)
+le = load(f"{data_path}/trained_nf1_model_label_encoder.joblib")
+model = load(f"{data_path}/trained_nf1_model.joblib")
+
+# Set the seed
```
Comment (jenna-tomkinson) [inline 2.evaluate_models/classify_genotypes/nbconverted/plate_4_sirna_genotype_probabilities.py]: Just curious: do the predicted probability values change every time you apply the model (kinda like UMAP embeddings), which is why you need to set a seed?

File: 2.evaluate_models/classify_genotypes/nbconverted/plate_4_sirna_genotype_probabilities.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+# ## Save siRNA genotype probabilities
+
+# In[6]:
+
+
+probabilitydf = pd.DataFrame(
+    {
+        f"probability {le.inverse_transform([1])[0]}":
+        model.predict_proba(plate4df[model.feature_names_in_])[:, 1]
```
Comment (jenna-tomkinson) [inline 2.evaluate_models/classify_genotypes/nbconverted/plate_4_sirna_genotype_probabilities.py]: What does this portion do? Will the column names be `probability WT` and `probability Null` with a space? Just curious to know what to expect in the parquet file.

**Review by jenna-tomkinson — APPROVED** [review]
LGTM! 🎉 I added a few clarifying questions, but feel free to merge after addressing. Super exciting stuff!

File: 2.evaluate_models/classify_genotypes/nbconverted/plate_4_sirna_genotype_probabilities.py (hunk 1 above)
Comment (MattsonCam) [reply 2.evaluate_models/classify_genotypes/nbconverted/plate_4_sirna_genotype_probabilities.py]: Oh no, the logistic regression is already parameterized. That was unused code, it's removed now!

File: 2.evaluate_models/classify_genotypes/nbconverted/plate_4_sirna_genotype_probabilities.py (hunk 2 above)
Comment (MattsonCam) [reply 2.evaluate_models/classify_genotypes/nbconverted/plate_4_sirna_genotype_probabilities.py]: Good point, I don't like that either, just changed with an underscore. There will only be one probability column. I think it is WT.

**Review by MattsonCam — COMMENTED** [review]
Thanks @jenna-tomkinson, merging now!

**Pull request merged**